### PR TITLE
Added session "What's new in HealthKit"

### PR DESCRIPTION
**Pull request by atereshkov**
## Checklist

- [x] I have read the [CONTRIBUTING.md](https://github.com/Blackjacx/WWDC/blob/master/CONTRIBUTING.md) file.
- [x] I have reviewed my own notes for typos, grammar and consistency (if you're submitting notes for a new session)

## Description
Added session "What's new in HealthKit"

### CODE REVIEW

File: summaries/2020/10182.md [hunk 1]
```diff
@@ -2,6 +2,36 @@
 
 https://developer.apple.com/wwdc20/10182
 
-Presenters: _Example Guy, Another Person_
+Presenter: _Netra Kenkarey_
 
-##### TO-DO! You can contribute to this session, please see [CONTRIBUTING.md](CONTRIBUTING.md)
+- **Symptoms**
+  - Developers can now track symptoms in HealthKit, read and write symptom samples
+  - HealthKit attempted to cover and track a wide range of symptoms (shortness of breath, sleep changes, appetite changes, fever, headache, etc)
```
Comment (Blackjacx) [inline summaries/2020/10182.md]:
```suggestion
  - HealthKit attempted to cover and track a wide range of symptoms (shortness of breath, sleep changes, appetite changes, fever, headache, etc.)
```

File: summaries/2020/10182.md [hunk 2]
```diff
@@ -2,6 +2,36 @@
...
+  - There are 13 symptom data types in HealthKit
+- **Electrocardiogram (ECG)**
+  - ECG samples will be available for reading in the latest version of iOS and watchOS
+  - An ECG sample can be read as an [`HKElectrocardiogram`](https://developer.apple.com/documentation/healthkit/hkelectrocardiogram) (it represents a waveform as a series of voltage values)
+  - ECG sample has important properties that describe the measurements
+  - [classification (`HKElectrocardiogram.Classification`)](https://developer.apple.com/documentation/healthkit/hkelectrocardiogram/3551981-classification)
+    - Apple Watch will give the result of the recording in the form of a classification
+    - The classification is divided into 2 types
+      - Sinus Rhithm (heart is beating in a uniform pattern)
```
Comment (Blackjacx) [inline summaries/2020/10182.md]:
```suggestion
      - Sinus Rhythm (heart is beating in a uniform pattern)
```

**Review by Blackjacx — APPROVED** [review]

**Review by Blackjacx — COMMENTED** [review]
Very nice 🎉 This is exactly how it should be 🙏 I fixed the typos right away. Hope it's okay for you :-)

**Comment by Blackjacx** [comment]
By the way, it is really good that you cover your information by links 🙏

**Comment by atereshkov** [comment]
@Blackjacx great!

**Pull request merged**